### ISSUE TITLE
how to save live photo

### ISSUE THREAD

**Issue by qyzhaojinxi** [versluis/Live-Photo-Converter#1]
hi,did you find any ways to save live photo?I have tried  a lot  ,but it didn't work.

**Comment by versluis** [comment]
Sadly not - I think we've all tried a lot, and really hard, but nobody seems to have had any success saving a Live Photo yet.

**Comment by qyzhaojinxi** [comment]
Right,I also tried a lot.I think maybe because the image and movie are not the same name and not in the same ablum.What do you think?

**Comment by versluis** [comment]
I don't think that's the problem: the UIImage could come from another source, like the Documents directory. I think there's some meta data missing somewhere, and iOS sees our fake Live Photo as invalid. The documentation states that if you turn a Live Photo into data and back again, the reconstructed version has to come from a "valid Live Photo". This leads me to believe that there's some additional check going on under the hood.

After all, if Apple wanted us to create Live Photos this way, they would have exposed this API publicly ;-) They want to keep this an exclusive iPhone 6s/6s Plus feature for now. Not even the iPad Pro can create Live Photos, and it has more than capable hardware.

Maybe in a later iOS release?

**Comment by qyzhaojinxi** [comment]
I found an app can do this.I can't understand how it works.https://itunes.apple.com/cn/app/livemaker-free-for-live-photo/id1053967828?mt=8

**Comment by qyzhaojinxi** [comment]
Hi,have you solved it yet?

**Comment by minjoongkim** [comment]
this project is success.
but swift..

https://github.com/genadyo/LivePhotoDemo
